python/mage/max_flow/bfs_weight_min_max.py

```diff
@@ -0,0 +1,36 @@
+import mgp
+
+
+def BFS_find_weight_min_max(start_v: mgp.Vertex, edge_property: str) -> mgp.Number:
+    """
+    Breadth-first search for finding the largest and smallest edge weight,
+    largest being used for capacity scaling, and smallest for lower bound
+
+    :param start_v: starting vertex
+    :param edge_propery: str denoting the edge property used as weight
+
+    :return: Number, the largest value of edge_property in graph
+    """
+
+    next_queue = [start_v]
```

Comment:
Consider using `collections.dequeu()` here since it should be more optimized for popping elements from start